Makefile

```diff
@@ -175,6 +176,9 @@ issue178_test: issues/issue178_test.o $(LIBOBJECTS) $(TESTHARNESS)
 issue200_test: issues/issue200_test.o $(LIBOBJECTS) $(TESTHARNESS)
 	$(CXX) $(LDFLAGS) issues/issue200_test.o $(LIBOBJECTS) $(TESTHARNESS) -o $@ $(LIBS)
 
+issue320_test: issues/issue320_test.o $(LIBOBJECTS) $(TESTHARNESS)
```

Comment:
@richcole-at-amazon the Makefile has been removed since you first submitted this and replaced with CMakeLists.txt (for cmake). Can you make a one-line addition to that file for this new test?